BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -28,6 +28,7 @@ public struct AccessibilityElement: Element {
     public var hint: String?
     public var identifier: String?
     public var traits: Set<Trait>
+    public var accessibilityFrameSize: CGSize?
```

Comment:
Maybe instead of an explicit size, a minimum size would be better? No strong opinions either way, but that feels less fragile than a fixed size; eg if the wrapped view changes due to a design change.

Reply:
I actually started with that but thought that wouldn't let you make a smaller rect in some cases.. might be worth adding an enum for different options in the future? But probably will just add this for now